arches/app/utils/data_management/resources/formats/rdffile.py

```diff
@@ -277,24 +277,13 @@ def __init__(self):
         self.resources = []
         self.resource = None
         self.use_ids = False
-        self.resource_model_root_classes = set()
```

Comment:
These features from the first implementation weren't being used, so I deleted them